infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -353,7 +366,8 @@ private void bootstrapPrimarySequencerServer() {
      * @return True if recovery was successful. False otherwise.
      */
     private boolean runRecoveryReconfiguration() {
-        Layout layout = new Layout(serverContext.getManagementLayout());
+        Layout layout = getManagementLayout();
+        Layout localRecoveryLayout = new Layout(layout);
         boolean recoveryReconfigurationResult = reconfigurationEventHandler
                 .recoverCluster(layout, getCorfuRuntime());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Parameter 1 to this call is marked "javax.annotation.Nonnull" but null could be passed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2637)